src/components/ha-push-notifications-toggle.js

```diff
@@ -88,7 +91,11 @@ class HaPushNotificationsToggle extends EventsMixin(PolymerElement) {
         browserName = "chrome";
       }
 
-      const name = prompt("What should this device be called ?");
+      const name = await showPromptDialog(this, {
+        title: "What should this device be called ?",
+        inputLabel: "Please enter a name for the device",
```

Comment:
Those texts should be localized.

Reply:
What would be the appropriate path for the texts?

Reply:
There is already a group "[...].profile.push_notifications" which I think makes sense. So perhaps "[...].profile.push_notifications.add_device.title" and "[...].profile.push_notifications.add_device.text"